openmdao/drivers/pyoptsparse_driver.py

```diff
@@ -32,10 +52,10 @@ def __init__(self):
         self.supports['inequality_constraints'] = True
         self.supports['equality_constraints'] = True
         self.supports['multiple_objectives'] = False
-        self.supports['two_sided_constraints'] = True
 
         # TODO: Support these
         self.supports['linear_constraints'] = False
+        self.supports['two_sided_constraints'] = False
```

Comment:
I was looking at the __init__ function, where the value was changed from True to False for 'two_sided_constraints'